examples/blog/invoiceGenerator/src/components/pdf/pdfLayout.tsx

```diff
@@ -12,12 +12,14 @@ type PdfProps = {
 };
 
 export const PdfLayout: React.FC<PdfProps> = ({ record }) => {
+    // eslint-disable-next-line
     const pdfExportComponent = useRef<any>();
 
     const handleExportWithComponent = () => {
         pdfExportComponent?.current?.save();
     };
 
+    // eslint-disable-next-line
     const total = record?.missions.reduce((prev: any, cur: any): any => {
         return prev + cur.day * cur.daily_rate;
     }, 0);
```

Comment:
```suggestion
    const total =
        record?.missions.reduce((prev, cur) => {
            return prev + cur.day * cur.daily_rate;
        }, 0) ?? 0;
```

No need for `any` here

Reply:
done